index.html

```diff
@@ -179,7 +179,7 @@ <h2>Contact Us</h2>
 
     </main>
 
-    <footer role="contentinfo">
```

Comment:
Comment: Removed 'role' attribute as it is unnecessary.

Reply:
thanks